Cargo.toml

```diff
@@ -77,10 +77,9 @@ default-features = false
 features = ["parsing"]
 
 [dependencies.clap]
-version = "3.2.20"
+version = "4.0.2"
```

Comment:
I think it would be nice to bump to latest already now (4.0.10 as of now)

Reply:
Done. (Note that the precise version is specified in the lockfile, not here)